### PR TITLE
Modest updates to README.md because 1.0 already happened

### PR DESCRIPTION
**Pull request by rfay**
- [x] I have reviewed the [Contributor Guidelines][contributor].
- [x] I have reviewed the [Code of Conduct][coc] and agree to abide by it

[contributor]: https://github.com/bats-core/bats-core/blob/master/docs/CONTRIBUTING.md
[coc]:         https://github.com/bats-core/bats-core/blob/master/docs/CODE_OF_CONDUCT.md

This PR just updates the README.md a bit based on the current status of the project. v1.0 happened some time ago, etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -451,40 +444,24 @@ See `docs/CHANGELOG.md`.
...
+
+[bats-orig]: https://github.com/sstephenson/bats
+[0360811]: https://github.com/sstephenson/bats/commit/03608115df2071fff4eaaff1605768c275e5f81f
+
+This [bats-core repo](https://github.com/bats-core/bats-core) is now the official Bats project.
+
 ### Why was this fork created?
 
-The original Bats repository needed new maintainers, and has not been actively
+The original Bats repository needed new maintainers, and had not been actively
```
Comment (cdevinesr) [inline README.md]: I would recommend dropping this particular change.  This adjustment to wording makes it sound like the original bats project is being actively maintained by the original maintainer again, when that is not the case.

Comment (sublimino) [reply README.md]: Although it's written in the past tense and sounds clear enough IMHO, perhaps "is no longer maintained" would dispel any ambiguity.

**Comment by sublimino** [comment]
Generally sounds good to me. If you wouldn't mind making that one change @rfay I'll merge. Thanks 🙏

**Comment by rfay** [comment]
Simplified to "The original Bats repository is no longer maintained and write access to it could not be obtained. This fork allowed ongoing maintenance and forward progress for Bats". Hope that works.

**Comment by sublimino** [comment]
LGTM, thanks for the changes @rfay  :+1:

File: README.md [hunk 2]
```diff
@@ -451,39 +444,20 @@ See `docs/CHANGELOG.md`.
...
-hindered progress severely.
+**Tuesday, September 19, 2017:** This was forked from [Bats][bats-orig] at
+commit [0360811][].  It was created via `git clone --bare` and `git push
+--mirror`. See the [Background](#background) section above for more information.
 
-### What's the plan and why?
+[bats-orig]: https://github.com/sstephenson/bats
+[0360811]: https://github.com/sstephenson/bats/commit/03608115df2071fff4eaaff1605768c275e5f81f
+
+This [bats-core repo](https://github.com/bats-core/bats-core) is now the official Bats project.
```
Comment (jasonkarns) [inline README.md]: I'm not sure anyone is in the position of claiming which project is "official" other than Sam himself. I think we should reword this to indicate it's the main community project or something?

File: README.md [hunk 3]
```diff
@@ -451,39 +444,20 @@ See `docs/CHANGELOG.md`.
...
-
-Doing it this way accomplishes a number of things:
-
-1. Removes the dependency on the original maintainer
-1. Enables collaboration and contribution flow again
-1. Allows the possibility of merging back to original, or merging from original
-   if or when the need arises
-1. Prevents lock-out by giving administrative access to more than one person,
-   increases transferability
+The original Bats repository is no longer maintained and write access to it could not be obtained. This fork allowed ongoing maintenance and forward progress for Bats.
```
Comment (jasonkarns) [inline README.md]: I'm also a little wary of the phrasing "is no longer maintained"... Again, there's a fine distinction between a project officially declared unmaintained by the owner, and one _appearing_ unmaintained from the outside.

Bats upstream is definitely the latter, but only questionably the former. I think we can word this better to be unambiguous without simply declaring the status of upstream by fiat.

Comment (btamayo) [reply README.md]: Good point, I'd agree with this.
